tools/panaroo/panaroo.xml

```diff
@@ -0,0 +1,156 @@
+            <param argument="--family_threshold" type="float" optional="true" value="0.7" label="Family Sequence Identity Level" help="Set the level at which Panaroo collapses genes into putative families. Default is 0.7."/>
+            <param argument="--len_dif_percent" type="float" optional="true" value="0.98" label="Length Difference Percent" help="Set the length difference percent for initial gene clustering. Default is 0.98."/>
+            <param name="merge_paralogs" type="boolean" optional="true" truevalue="--merge_paralogs" falsevalue="" label="Merge Paralogs" help="Merging paralogs can be enabled. Disabled by default."/>
+        </section>
+        <!--Refind-->
+        <section name="refind" title="Refind"  expanded="false">
+            <param argument="--search_radius" type="integer" optional="true" value="5000" label="Refinding Search Radius" help="Set radius for the search in nucleotides for refinding genes. Default is 5000."/>
+            <param argument="--refind_prop_match" type="float" optional="true" value="0.5" label="Refinding Proportion Match" help="Set the proportion of the missing gene to consider as a match for refinding. Default is 0.5."/>
+            <param argument="--refind-mode" type="select" optional="true" label="Refind Mode" help="Set the stringency mode at which to re-find genes.">
+                <option value="default" selected="True">Default</option>
+                <option value="strict">Strict</option>
+                <option value="off ">Off</option>
+            </param>
+        </section>
+        <!--GRAPH CORRECTION -->
+
+        <!--Gene Alignment-->
+        <section name="gene_alignment" title="Gene Alignment"  expanded="false">
+            <param name="alignment" type="select" optional="true" label="Alignment" help="Run MSA either for the core genome or for every gene cluster.">
+                <option value="core" selected="True">Core genome alignment</option>
+                <option value="pan">Pan-genome alignment</option>
+            </param>
+            <param argument="--aligner" type="select" optional="true" label="Alignment method" help="Choose an aligner from either PRANK, Clustal or the default, MAFFT.">
+                <option value="mafft" selected="True">MAFFT</option>
+                <option value="clustal">Clustal</option>
+                <option value="prank">PRANK</option>
+                <option value="none">None</option>
+            </param>
+        </section>
+    </inputs>
+
+    <outputs>
+        <data format="csv" name="gene_p_a" label="${tool.name} on ${on_string} Gene Presence Absence" from_work_dir="out/gene_presence_absence.csv"/>
+        <data format="fasta" name="core_gene_aln" label="${tool.name} on ${on_string} Core Gene Alignment" from_work_dir="out/core_gene_alignment.aln"/>
```

Comment:
Please add the following files to output as well:

├── combined_DNA_CDS.fasta
├── combined_protein_cdhit_out.txt
├── combined_protein_cdhit_out.txt.clstr
├── combined_protein_CDS.fasta
├── final_graph.gml
├── gene_data.csv
├── gene_presence_absence.csv
├── gene_presence_absence_roary.csv
├── gene_presence_absence.Rtab
├── pan_genome_reference.fa
├── pre_filt_graph.gml
├── struct_presence_absence.Rtab
└── summary_statistics.txt

For gml, file format you can specify datatype as text for now

Reply:
I can't find some of these Output files in here: https://gthlab.au/panaroo/#/gettingstarted/output